tests/test_core.py

```diff
@@ -321,3 +321,21 @@ def test_gen_XTC_calcOP(self, tmpdir):
         ref_file_apineiro = path_data / "ref_2POPC_traj.apineiro.out"
         assert filecmp.cmp(test_file_jmelcr, ref_file_jmelcr)
         assert filecmp.cmp(test_file_apineiro, ref_file_apineiro)
+
+
+
+    def test_check_def_file(self):
+        """
+        Test for is_allHs_present()
+        """
```

Comment:
Could you please check/enforce [PEP257](https://www.python.org/dev/peps/pep-0257/) in your test scripts? For instance with `pydocstyle`.

With [one-line docstring](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings), you should have:
```
"""Test for is_allHs_present()."""
```

Reply:
We could do this later though...

Reply:
Damn, I didn't know about those "one-line docstring" :)

Maybe we can open an issue about the PEP257 check since its scope is broader than this PR.

Reply:
OK.